tests/functional/test_lex.py

```diff
@@ -45,7 +45,7 @@ def test_unsigned_payload(self):
         # signature. The signature will be part of the authorization header.
         # Since we don't have direct access to the payload signature,
         # we compare the authorization instead.
-        authorization = request.headers.get('authorization')
+        authorization = request.headers.get('Authorization')
```

Comment:
Why did these change?  Are we not pulling in a case insensitive dict anymore?

Reply:
We weren't originally, but I've since added a case insensitive implementation. I'll revert these changes to reduce noise.